use std::io::Error;

pub struct Data {

}

fn save_data(day: u32, part_2: bool) -> Result<Data, Error> {

}

